src/io/config.cpp

```diff
@@ -42,6 +42,19 @@ std::unordered_map<std::string, std::string> Config::Str2Map(const char* paramet
   for (auto arg : args) {
     KV2Map(&params, Common::Trim(arg).c_str());
   }
+  // define verbosity and set logging level
+  int verbosity = Config().verbosity;
```

Comment:
Shouldn't we update the following function as well?
https://github.com/microsoft/LightGBM/blob/1cc9f9dcee15586aefdd8271a1c04ad010d3c53a/src/application/application.cpp#L50

Reply:
I believe if I move the logic into `KeyAliasTransform` this function will work as expected as well, except for the duplicate argument warnings.

Reply:
I think it's good idea to move the logic into `KeyAliasTransform`.